src/api/publications.ts

```diff
@@ -11,3 +20,25 @@ export const createPublication = (
         detail: entityDescription ?? null,
     });
 };
+
+export const getPublicationById = async (pubId: string) => {
+    const data = await supabaseRetry(
+        () =>
+            supabaseClient
+                .from(TABLES.PUBLICATIONS)
+                .select(JOB_STATUS_COLUMNS)
+                .eq('id', pubId),
+        'getPublicationById'
+    ).then(
+        handleSuccess<
+            {
+                job_status: { type: string };
+                logs_token: string;
+                id: string;
+            }[]
```

Comment:
Can you toss this into an `interface`... probably in a shared spot near the columns... but not super opinionated on where it should live.